lib/easy_meli.rb

```diff
@@ -25,12 +25,12 @@ def self.create_token(code, redirect_uri, logger: nil)
     EasyMeli::AuthorizationClient.create_token(code, redirect_uri, logger: logger)
   end
 
-  def self.refresh_token(refresh_token, logger: nil)
-    EasyMeli::AuthorizationClient.refresh_token(refresh_token, logger: logger)
+  def self.access_token(refresh_token, logger: nil)
+    EasyMeli::AuthorizationClient.access_token(refresh_token, logger: logger)
   end
 
   def self.api_client(access_token: nil, refresh_token: nil, logger: nil)
-    access_token = self.refresh_token(refresh_token, logger: logger) if refresh_token
+    access_token ||= self.access_token(refresh_token, logger: logger) if refresh_token
```

Comment:
We noticed that if you pass the access token and refresh token the first was ignored. This adds the `||=` in order to keep the existing access token.